src/main/java/step2/Calc.java

```diff
@@ -0,0 +1,64 @@
+package step2;
+
+import java.util.Scanner;
+import java.util.Stack;
+import java.util.regex.Pattern;
+
+public class Calc {
+	private final String ONLY_NUMBER = "^[0-9]+$";
+	private final String ONLY_OPERATOR = "^|\\*|\\/|\\+|-|$";
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		String str = scanner.nextLine();
+		scanner.close();
+
+		System.out.println("result : " + new Calc().getResult(str));
+	}
+
+	public String getResult(String str) {
+		if(this.isEmpty(str)) throw new IllegalArgumentException("입력 값이 null이거나 빈 공백 문자");
+
+		String[] values = str.split(" ");
+		Stack<String> expression = new Stack<>();
+		for(String value : values) {
+			expression.push(value);
+			if(value.matches(ONLY_NUMBER) && expression.size() == 3) {
```

Comment:
다음 자동차 경주 미션부터 다음과 같은 프로그래밍 요구사항이 존재하는데요.
- indent(인덴트, 들여쓰기) depth를 2를 넘지 않도록 구현한다. 1까지만 허용한다.
    - 예를 들어 for문 안에 if문이 있으면 들여쓰기는 2이다.
    - 힌트: indent(인덴트, 들여쓰기) depth를 줄이는 좋은 방법은 함수(또는 메소드)를 분리하면 된다.

한 메서드 안에 중첩된 반복 혹은 제어 구조가 있다면 다단계의 추상화를 코드로 짠 것이며, 메서드가 한 가지 이상의 일을 하고 있다는 신호입니다. indent를 1로 줄이도록 메서드 분리를 해보면 어떨까요 :) ?

```suggestion
		for (String value : values) {
		    expression.push(value);
		    if (value.matches(ONLY_NUMBER) && expression.size() == 3) {
```